### PR TITLE
Hercules MP3 e2: Add doc for new mapping

### PR DESCRIPTION
**Pull request by 0xYoan**
Here's a PR of the documentation for the new mapping of Hercules DJ Control MP3e2 controller.
New mapping PR link : https://github.com/mixxxdj/mixxx/pull/3857

Manual PR: https://github.com/mixxxdj/manual/pull/387

### CODE REVIEW

File: source/hardware/controllers/hercules_djcontrol_mp3_e2.rst [hunk 1]
```diff
@@ -3,589 +3,123 @@ Hercules DJControl MP3 e2 / MP3 LE / Glow
 
 The Hercules DJ Control MP3 e2, MP3 LE, and Glow are USB controllers that have identical controls and share the same mapping.
 These controllers do not have a built in sound card, so a :ref:`splitter cable <hardware-splitter-cables>` or :ref:`separate audio interface <hardware-audio-interfaces>` is recommended for use with it.
-The mapping is included in Mixxx and allows you to manipulate 4 decks, 2 at a time, switching Deck A (left) between Channel 1 and 3 and Deck B (right) between Channel 2 and 4.
 
-.. versionadded:: 1.11
+.. versionadded:: 2.3
```
Comment (Holzhaus) [inline source/hardware/controllers/hercules_djcontrol_mp3_e2.rst:7]: The mapping was included in 1.11.
```suggestion
.. versionadded:: 1.11
.. versionchanged:: 2.3.0
   Please describe give a short description of user-visible changes in a sentence or two.
```

File: source/hardware/controllers/hercules_djcontrol_mp3_e2.rst [hunk 2]
```diff
@@ -3,589 +3,123 @@ Hercules DJControl MP3 e2 / MP3 LE / Glow
...
+| Toggle playlist selection                                                  | Folder     | 6      |
++----------------------------------------------------------------------------+------------+--------+
+| Toggle track selection                                                     | Files      | 10     |
++----------------------------------------------------------------------------+------------+--------+
+| Go one playlst/track up                                                    | Up arrow   | 8      |
++----------------------------------------------------------------------------+------------+--------+
+| Go one playlst/track down                                                  | Down arrow | 8      |
++----------------------------------------------------------------------------+------------+--------+
+| Loads the currently highlighted track into the corresponding deck (A or B) | Load A/B   | 18     |
++----------------------------------------------------------------------------+------------+--------+
```
Comment (Holzhaus) [inline source/hardware/controllers/hercules_djcontrol_mp3_e2.rst:43]: Please use the simpler table formatting (as in `=`) and ideally use the following column order: Number, Control, Function

Also, if a control is *labeled* on the controller itself, please use the hwlabel directive, e.g.:

    :hwlabel:`CUE`

File: source/hardware/controllers/hercules_djcontrol_mp3_e2.rst [hunk 3]
```diff
@@ -3,589 +3,123 @@ Hercules DJControl MP3 e2 / MP3 LE / Glow
...
-Select prev/next track                                                      Supershift + Deck B Jog Wheel 16
-Loads the currently highlighted track into the corresponding deck (A or B)  Load A/B                      18
-Loads the current highlighted track into the corresponding sampler (1 or 2) Shift + Load A/B              18
-=========================================================================== ============================= ======
-
-Master/Headphones/Microphone
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-.. figure:: ../../_static/controllers/hercules_djcontrol_mp3_e2_masterheadmicro.jpg
+.. figure:: ../../_static/controllers/hercules_djcontrol_mp3_e2_mapping.png
```
Comment (Holzhaus) [inline source/hardware/controllers/hercules_djcontrol_mp3_e2.rst:15]: Do you habe an SVG file for this? That would be even better than a JPG.

File: source/hardware/controllers/hercules_djcontrol_mp3_e2.rst [hunk 4]
```diff
@@ -3,589 +3,123 @@ Hercules DJControl MP3 e2 / MP3 LE / Glow
...
-Toggles deck output to the headphones monitor on/off              Headphone monitor         20
-================================================================= ========================= ======
+Mapping description (by function)
+---------------------------------
 
-Sampler
+Library
 ^^^^^^^
 
-.. figure:: ../../_static/controllers/hercules_djcontrol_mp3_e2_sampler.jpg
```
Comment (Holzhaus) [inline source/hardware/controllers/hercules_djcontrol_mp3_e2.rst:119]: Please delete those images if they are not used anymore.

File: source/hardware/controllers/hercules_djcontrol_mp3_e2.rst (hunk 3 above)
Comment (0xYoan) [reply source/hardware/controllers/hercules_djcontrol_mp3_e2.rst:18]: Sorry but I don't have an SVG file for this.

Comment (Holzhaus) [reply source/hardware/controllers/hercules_djcontrol_mp3_e2.rst:18]: Where did you get the image from. Sometimes it 's possible to open the owner manual PDF with Inkscape.

Comment (0xYoan) [reply source/hardware/controllers/hercules_djcontrol_mp3_e2.rst]: The image was added by the last contributor before me. I think I've succeed to add an SVG file for it. Learned something new, that's cool :)

File: .gitignore [hunk 5]
```diff
@@ -3,5 +3,8 @@
 *.pyc
 
 # Generated during a regular build
-build
```
Comment (Holzhaus) [inline .gitignore:6]: This is the standard build directory, please don't remove this.

File: .gitignore [hunk 6]
```diff
@@ -3,5 +3,8 @@
...
+html
```
Comment (Holzhaus) [inline .gitignore:6]: You can of course use a different name, but let's not add it to `.gitignore`.

File: .gitignore [hunk 7]
```diff
@@ -3,5 +3,8 @@
...
 source/locale/pot/.doctrees
+
+# VSCode generated files
+.vscode
```
Comment (Holzhaus) [inline .gitignore:10]: Please don't add it here. We cannot add entries for every possible IDE/editor out there. Add it to you global gitingore instand: https://openqa-bites.github.io/posts/2021-04-23-global-gitignore/

**Review by Holzhaus — COMMENTED** [review]
Please revert the gitignore changes. :wink:

**Review by Holzhaus — APPROVED** [review]

**Comment by uklotzde** [comment]
Note: Commits should be squashed before merging!

**Comment by uklotzde** [comment]
@Holzhaus Manual PR has been approved. Squash and merge?

**Comment by uklotzde** [comment]
@Holzhaus Manual PR has been approved. Squash and merge?

**Comment by ronso0** [comment]
squashed & merged as suggested, and now I got this pre-commit fail notification even though here it reads "5 checks passed"?
https://github.com/mixxxdj/manual/actions/runs/1271521821

**Pull request merged**